config/sarc-dev.json

```diff
@@ -65,7 +105,30 @@
             "prometheus_url": null,
             "prometheus_headers_file": null,
             "start_date": "2022-04-01",
-            "nodes_info_file": "secrets/nodes_graham.txt"
+            "nodes_info_file": "secrets/nodes_graham.txt",
+            "gpus_per_nodes": {
+                "gra{{[828-987]}}": {
+                    "p100": "NVIDIA P100-12G PCIe"
+                },
+                "gra{{[1147-1153]}}": {
+                    "v100": "NVIDIA V100-16G PCIe"
+                },
+                "gra{{[1154-1189]}}": {
+                    "t4": "NVIDIA T4-16G PCIe"
+                },
+                "gra{{[1337-1338]}}": {
+                    "v100": "NVIDIA V100-32G PCIe"
+                },
+                "gra1342": {
+                    "a100": "NVIDIA A100 PCIe"
+                },
+                "gra{{[1361-1362]}}": {
+                    "a100": "NVIDIA A100 PCIe"
+                },
+                "gra{{[1363-1373]}}": {
+                    "a5000": "NVIDIA A5000-24G PCIe"
+                }
+            }
```

Comment:
I guessed that all of the GPUs where PCIe connected? The info doesn't seams to be present in the docs : https://docs.alliancecan.ca/wiki/Graham. Maybe it's cleaner to simply omit the socket for the non v100, a100 GPUs?